openedx/features/edly/utils.py

```diff
@@ -1,6 +1,13 @@
+import logging
+
 import jwt
+import waffle
 from django.conf import settings
+from django.forms.models import model_to_dict
+from openedx.features.edly.models import EdlySubOrganization
```

Comment:
nit:  add a new line above local imports so this its separate from the group of django+third party imports